src/main/java/com/onarandombox/MultiverseCore/config/MVCoreConfigNodes.java

```diff
@@ -113,6 +112,21 @@ private <N extends Node> N node(N node) {
             .name("first-spawn-location")
             .build());
 
+    public final ConfigNode<String> JOIN_DESTINATION = node(ConfigNode.builder("spawn.always-spawn-destination", String.class)
+            .comment("")
+            .comment("Sets the destination that Multiverse will use to spawn players on every login")
+            .comment("Set blank to disable")
+            .defaultValue("")
+            .name("always-spawn-destination")
+            .build());
+
+    public final ConfigNode<Boolean> ENABLE_JOIN_DESTINATION = node(ConfigNode.builder("spawn.enable-always-spawn-destination", Boolean.class)
```

Comment:
I prefer the order where this boolean to enable is above the join-location string, just like `first-spawn-override`